scripts/review.js

```diff
@@ -35,7 +35,7 @@ module.exports = (robot) => {
       'url'
     ])
     .then((response) => {
```

Comment:
@paroxp if you are thinking about changing code in this file, can you change this to use `async/await`? It's kind of the standard that I'm trying to set in the codebase. I'm sorry I didn't leave this comment on the original code review, we use promises at work because we can't use anything newer so rolled right over it in the code review.